elixir/apps/domain/lib/domain/auth/adapter/openid_connect/directory_sync.ex

```diff
@@ -285,17 +285,17 @@ defmodule Domain.Auth.Adapter.OpenIDConnect.DirectorySync do
            plan: {identities_insert_ids, identities_update_ids, identities_delete_ids},
            inserted: identities_inserted,
            updated: identities_updated,
-           deleted: identities_deleted
+           deleted_count: identities_deleted_count
          },
          %{
            plan: {groups_upsert_ids, groups_delete_ids},
            upserted: groups_upserted,
-           deleted: groups_deleted
+           deleted_count: groups_deleted_count
          },
          %{
            plan: {memberships_insert_tuples, memberships_delete_tuples},
            inserted: memberships_inserted,
-           deleted_stats: {deleted_memberships_count, _}
+           deleted_count: {memberships_deleted_count, _}
```

Comment:
Just double-checking that pattern-matching against a tuple was intended here since the other counts didn't have it.